cmake/external/ngraph.cmake

```diff
@@ -37,7 +37,7 @@ INCLUDE(GNUInstallDirs)
 INCLUDE(ExternalProject)
 
 SET(NGRAPH_PROJECT         "extern_ngraph")
-SET(NGRAPH_GIT_TAG         "096ad6ef0c04d57db1522940dbdf9a0652768065")
+SET(NGRAPH_GIT_TAG         "4ec94acc11084a5d53418f565529310fa584899a")
```

Comment:
why change this？

Reply:
The ngraph::op::ScatterAdd (used in this lookup_table_op) is supported with new ngraph version.